readers/influxdb/messages.go

```diff
@@ -7,10 +7,10 @@ import (
 	"strconv"
 	"strings"
 	"time"
-
+	
```

Comment:
Looks like you are using spaces to complete the tabulation. Can you please remove it here and in all other places.